lms/djangoapps/badges/api/views.py

```diff
@@ -0,0 +1,135 @@
+"""
+API views for badges
+"""
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey
+from rest_framework import generics
+from rest_framework.exceptions import APIException
+
+from openedx.core.djangoapps.user_api.permissions import is_field_shared_factory
+from openedx.core.lib.api.authentication import (
+    OAuth2AuthenticationAllowInactiveUser,
+    SessionAuthenticationAllowInactiveUser
+)
+from badges.models import BadgeAssertion
+from .serializers import BadgeAssertionSerializer
```

Comment:
I think we're supposed to avoid relative imports unless there's a good reason to have them